### PR TITLE
Converting2 json proccesing

### PR DESCRIPTION
**Pull request by ArastoSahbaei**

### CODE REVIEW

File: src/main/java/com/communityratesgames/model/GameModel.java [hunk 1]
```diff
@@ -1,23 +1,126 @@
...
+
+
+         //   gameModel.setCompany(json.getString("company"));
+           // gameModel.setPlatforms(herp);
+            return gameModel;
+        }
+        catch(Exception e) {
+            e.printStackTrace();
+            return null;
+        }
```
Comment (Hanicef) [inline src/main/java/com/communityratesgames/model/GameModel.java:67]: Catch more specific exceptions than just `Exception`. It's a bad habit that we must break out of.
Judging from your case, you would want to catch `IOException` and `JsonProcessingException` (See https://fasterxml.github.io/jackson-databind/javadoc/2.0.0/com/fasterxml/jackson/databind/ObjectMapper.html#readTree(byte[])).

File: src/main/java/com/communityratesgames/model/GameModel.java [hunk 2]
```diff
@@ -1,23 +1,126 @@
...
+
+        return object;
+    }
+
+
+    public GameModel test(String input) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<String> list = new ArrayList<>();
+        try {
+            JsonNode root = objectMapper.readTree(input);
```
Comment (Hanicef) [inline src/main/java/com/communityratesgames/model/GameModel.java:47]: Where does `input` come from? I can't find the definition of it anywhere.

File: src/main/java/com/communityratesgames/model/GameModel.java [hunk 3]
```diff
@@ -1,23 +1,126 @@
...
+
+
+            public GameModel(Game game) {
+            this.title = game.getTitle();
+            this.company = game.getCompany().getCompanyName();
+            this.platforms = game.getPlatforms().stream().map(Platform::getName).collect(Collectors.toList());
+            this.verified = game.isVerified();
+            this.averageRating = game.getAverageRating();
+        }
+    }
```
Comment (Hanicef) [inline src/main/java/com/communityratesgames/model/GameModel.java:96]: Something's not right with the indentation here; it took me a while to wrap my head around it and understand that it belongs to the `GameModel` class.

File: src/main/java/com/communityratesgames/domain/Game.java [hunk 4]
```diff
@@ -1,42 +1,38 @@
...
 import java.util.List;
-import java.util.ArrayList;
-
-import com.communityratesgames.model.GameModel;
-import com.communityratesgames.model.PlatformModel;
 
 @Entity
 @Table(name = "game_entity")
-public class Game {
+public class Game implements Serializable {
```
Comment (Hanicef) [inline src/main/java/com/communityratesgames/domain/Game.java:10]: We should do that with all the classes. `Serializable` doesn't even have any definition requirements.
Maybe starting an issue about it is a good idea.

**Review by Hanicef — CHANGES_REQUESTED** [review]
Has a few things that needs addressing, but most importantly, why did you remove `api.service.ts` and `url.service.ts`? I can only see reason why is should *not* be removed.

**Comment by Ivefan** [comment]
Arasto forgot to close this after making a new PR on relevant changes from this PR

**Pull request closed without merging**